packages/commonwealth/client/scripts/views/components/SublayoutHeader/useUserMenuItems.tsx

```diff
@@ -36,6 +46,9 @@ export const handleLogout = async () => {
     await axios.get(`${app.serverUrl()}/logout`);
     await initAppState();
     await resetWalletConnectSession();
+    for (const signer of getSessionSigners()) {
+      signer.target.clear();
+    }
```

Comment:
The session remains expired for all addresses other than the active one, in case when multiple addresses are connected to a community

1. Connect multiple addresses to a community
2. Expire sessions manually -> (`localStorage.clear()`)
3. Logout
4. Login
5. The session is still expired for all addresses, other than the active one.

This was with an Ethereum community

Reply:
I think that's intended behavior - localStorage.clear() clears all sessions, so no session keys are retained for other addresses.

Reply:
Do we not generate new session keys on login?

Reply:
Only for the active address - other addresses have to log in again (fixing that).